config/environment.js

```diff
@@ -195,6 +195,7 @@ module.exports = function (environment) {
     'github-apps': false,
     'enable-assembla-login': false,
     'enable-bitbucket-login': false,
+    'enable-gitlab-login': false,
```

Comment:
I think we were hoping to start eliminating `enable-` from the beginning of flag names. All feature flags are about enabling or disabling something, so it is somewhat redundant. And the `gitlab-login` flag already existed below.

Reply:
You are right Sophie. As it's now only for testing purposes I will leave it as it is and we will clean it before GL release.

Reply:
Cool, that works for me :)